mlir/test/Dialect/Vector/vector-reduce-to-contract.mlir

```diff
@@ -116,6 +156,39 @@ func.func @contract_broadcast_unit_dim_reduction(%arg0 : vector<8x4xi32>, %arg1
     return %result : vector<8x8xi32>
 }
 
+// -----
+
+// Same as above, but with a mask
+
+#map0 = affine_map<(d0, d1, d2, d3) -> (d0, d1, d3)>
+#map1 = affine_map<(d0, d1, d2, d3) -> (d0, d2, d3)>
+#map2 = affine_map<(d0, d1, d2, d3) -> (d1, d2)>
+
+// CHECK-DAG: #[[$map0:.*]] = affine_map<(d0, d1, d2) -> (d0, d2)>
+// CHECK-DAG: #[[$map1:.*]] = affine_map<(d0, d1, d2) -> (d1, d2)>
+// CHECK-DAG: #[[$map2:.*]] = affine_map<(d0, d1, d2) -> (d0, d1)>
```

Comment:
I'd use uppercase for consistency.

```suggestion
// CHECK-DAG: #[[$MAP0:.*]] = affine_map<(d0, d1, d2) -> (d0, d2)>
// CHECK-DAG: #[[$MAP1:.*]] = affine_map<(d0, d1, d2) -> (d1, d2)>
// CHECK-DAG: #[[$MAP2:.*]] = affine_map<(d0, d1, d2) -> (d0, d1)>
```

Reply:
I agree with the suggestion (thanks!) and will update this accordingly.

Note that currently this file prefers lower-case: https://github.com/llvm/llvm-project/blob/main/mlir/test/Dialect/Vector/vector-reduce-to-contract.mlir#L3

Let me update all maps within this PR - just to avoid creating a dedicated PR. If you think that the noise level is too high, I am happy to extract that.